src/main.cpp

```diff
@@ -8,13 +8,15 @@
 #include "ComponentTester.h"
 #include "Laser.h"
 #include "Properties.h"
+#include "module_PI/Arduino_Uno/SerialComm.h"
 
 Alarm* alarm;
+SerialComm* serialComm;
+int iterations = 0;
```

Comment:
Oh, nope. I must have used this for debugging purposes and forgot to take it out. Eclipse is usually good at telling me that kind of stuff. I'll fix it.